Sources/ProjectSpec/AggregateTarget.swift

```diff
@@ -1,5 +1,6 @@
 import Foundation
 import JSONUtilities
+import struct PathKit.Path
```

Comment:
Are there any naming conflicts here, that don't allow importing the whole module?

Reply:
No, just me letting NSHipster articles get into my head 😅